### ISSUE TITLE
Timing diagram lags behind the actual status

### ISSUE THREAD

**Issue by LeeTL1220** [broadinstitute/cromwell#1497]
I am running a workflow that will cache hit every job.  Given the log dump on the command line, that seems to be happening.  However, the timing diagram is not updating.  All I see is the first job and the timing diagram has been in that state for over ten minutes.

**Comment by LeeTL1220** [comment]
Desired behavior is to decrease this latency.

**Comment by geoffjentry** [comment]
@LeeTL1220 does it lag behind metadata or reality? if it is matching metadata that's life in the big city ;) If it's lagging that I agree that seems weird.

Also, if it's matching metadata your issue is really that the metadata lag is longer than you'd like. At that point it becomes a question as to whether it's longer than we believe it to be (which is quite possible)

**Comment by LeeTL1220** [comment]
Metadata is lagging reality by quite a bit.  Timing is just reflecting
that.  From my (I.e. User) perspective, it is hard to differentiate between
a timing versus metadata lag.

**Comment by geoffjentry** [comment]
@LeeTL1220 FWIW all endpoints that provide intel on the system (except for engine/stats) are going through metadata which intentionally lags behind reality, hooray eventual consistency. 

That said, I do believe there's something going on where it's a longer delay than we intended. Note that "we intended" doesn't mean 2 seconds, we expect that it'll always be a few minutes behind reality and that's just going to be life. However the times you've reported don't seem right.

I consider this a dupe of #1488 as AFAIK that was the first time you've reported the potentially overly slow metadatafication of things, closing this as this is just another symptom of the same problem.
